### PR TITLE
Yo check this out!!!

### PR DESCRIPTION
**Pull request by bluestar-b**

### CODE REVIEW

File: src/components/navbar.svelte [hunk 1]
```diff
@@ -10,7 +10,9 @@
 
 	<div class="text-md ml-auto flex space-x-2 font-bold">
 		<div>
-			<a href="https://github.com/notmycode-labs">GitHub</a>
+			<a href="https://github.com/notmycode-labs" target="_blank" rel="noopener noreferrer"
+				>GitHub</a
+			>
```
Comment (UnitieG) [inline src/components/navbar.svelte:15]: I don't think that suppose to be there.

Comment (bluestar-b) [reply src/components/navbar.svelte:15]: that cause by prettier formatter

> I don't think that suppose to be there.

**Comment by blueskychan-dev** [comment]
LGTM!!!

**Comment by UnitieG** [comment]
LGBTQ

**Comment by blueskychan-dev** [comment]
I gonna merge this, i think this is a good update.

**Pull request merged**